torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -373,12 +373,12 @@ def aot_dispatch_autograd(
                 colored=True,
             ),
         )
-        joint_graph_str = fx_g.print_readable(
+        joint_graph_str_fn = lambda: fx_g.print_readable(
```

Comment:
we have a `LazyString` class we use in a few other places to lazily materialize strings for logging - maybe we should use it here for consistency (to be fair it is a pretty tiny wrapper) https://github.com/pytorch/pytorch/blob/main/torch/_logging/_internal.py#L1083

Reply:
Yeah in this case since we're passing a lambda anyway I figured making it lazy string was literally more overhead

Reply:
You could technically just pass a functools.partial :P